frontend/src/components/Ranking/Ranking.jsx

```diff
@@ -40,37 +45,48 @@ export default function Ranking() {
-                    return (
-                        <div key={stock.ticker} className="stock-margin">
-                            <Link
-                                key={stock.name}
-                                to={`/search/stocks/${stock.ticker}`}
-                                className="stock-link"
-                            >
-                                <div className="stock">
-                                    <div className="stock-rank">
-                                        <p>{index + 1}</p>
-                                    </div>
-                                    <img
-                                        src={stock.logo}
-                                        alt={`This is a logo of ${stock.name}.`}
-                                    />
-                                    <div className="stock-info">
-                                        <p>
-                                            {stock.name} ({stock.ticker})
-                                        </p>
-                                    </div>
-                                </div>
-                            </Link>
-                        </div>
-                    );
-                })}
-            </div>
+        <div className="d-flex flex-column align-items-center justify-content-center">
+            <h1 className="mt-4">Popular Stocks</h1>
+            <p className="mb-4">
+                Recommended based on your profile and history.
+            </p>
+            {stocksRanking?.slice(0, stocksToShow).map((stock, index) => (
+                <div className="mb-4">
+                    <Link
+                        to={`/search/stocks/${stock.ticker}`}
+                        className="stock-link"
+                    >
+                        <Card key={stock.id} className="stock-rank-card">
+                            <div>
+                                <Card.Img variant="top" src={stock.logo} />
+                                <p
+                                    className={`stock-rank fs-3 ${
+                                        index === 0 ? "text-primary" : ""
+                                    }`}
+                                >
+                                    {index + 1}
+                                </p>
+                            </div>
+                            <Card.Body>
+                                <Card.Text className="text-center">
+                                    {stock.name} ({stock.ticker})
+                                </Card.Text>
+                            </Card.Body>
+                        </Card>
+                    </Link>
+                </div>
+            ))}
+            {stocksRanking?.length > stocksToShow ? (
+                <Button
+                    className="mb-5 mt-4"
+                    variant="outline-primary"
+                    onClick={() =>
+                        setStocksToShow((prev) => prev + STOCKS_TO_SHOW)
```

Comment:
This is not a very performant load more mechanism, since all the data is being sent to the client. The client is simply "rendering more", but everything has been loaded from the beginning. 

We need further discussion on this.